jme3-dark-laf/src/org/jme3/netbeans/plaf/darkmonkey/DarkMonkeyLookAndFeel.java

```diff
@@ -395,4 +401,29 @@ public String getThemeProperty(String name) {
     public String getDefaultThemeProperty(String name, String defaultValue) {
         return propertiesMap.get("blue").getProperty(name, defaultValue);
     }
+
+    private void applyDejaVuFontIfSupported(NimRODTheme theme) {
+        if (!userLangIsCJK()) {
+            theme.setFont(DarkMonkeyLookAndFeel.DEJA_VU_SANS_FONT);
+        }
+    }
+
+    /**
+     * DejaVu doesn't support CJK glyphs
+     * @return true if user locale is "ja" or "ko" or "zh"
+     */
+    private Boolean userLangIsCJK() {
+        // https://stackoverflow.com/a/18065154/2565527
+        // >The status document about language coverage indicate that's there is no glyph for CJK:
+        //
+        // >ja Japanese (0/6537) (0/6537) (0/6537)
+        // >ko Korean (0/2443) (0/2443) (0/2443)
+        // >zh-cn Chinese (simplified) 0% (2/6765) 0% (2/6765) 0% (2/6765)
+        // >zh-hk Chinese Hong Kong Supplementary Character Set (0/2213) (0/2213) (0/2213)
+        // >zh-mo Chinese in Macau (0/2213) (0/2213) (0/2213)
+        // >zh-sg Chinese in Singapore 0% (2/6765) 0% (2/6765) 0% (2/6765)
+        // >zh-tw Chinese (traditional) (0/13063) (0/13063) (0/13063)
+        String lang = System.getProperty("user.language");
+        return lang.matches("ja|ko|zh");
```

Comment:
Is something like startsWith required for Chinese or maybe `^zh-.*$` or something?

Reply:
I don't think so. Because the language code and country code is different property.
For example my default java language code and country code setting are 
```
-Duser.language=ja -Duser.country=jp
```

UI should follow user's property of `user.language`.

Reply:
I changed my mind, there are some minor patterns (can't detect supported or not by only language code).
It looks good to detect with both language code and country code.

```
mn-cn  Mongolian in China                            (0/130)            (0/130)            (0/130)      
mn-mn  Mongolian in Mongolia                    100% (70/70)       100% (70/70)       100% (70/70)      
```